.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ executors:
 
   container-base:
     docker:
-      - image: golang:1.14
+      - image: golang:1.15
```

Comment:
Do we also want to rebuild the CI images? It should pick the latest go version automatically but we have to bump/push a new version.

Reply:
done